source/fare/fare.cpp

```diff
@@ -138,27 +153,44 @@ results Fare::compute_fare(const routing::Path& path) const {
     labels[0].push_back(Label());
     size_t section_idx(0);
 
-    for (const auto& item : path.items) {
+    for (const navitia::routing::PathItem& item : path.items) {
```

Comment:
minor: `auto` was fine, no?

Reply:
yes it was, being explicit just help for code navigation